### ISSUE TITLE
x/pkgsite: Deprecation annotation for packages isn't displayed

### ISSUE THREAD

**Issue by johannes-riecken** [golang/go#66574]
### What is the URL of the page with the issue?
 
 https://pkg.go.dev/google.golang.org/api@v0.171.0/sql/v1beta4
 
### What is your user agent?
 

 
 [Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:99.0) Gecko/20100101 Firefox/99.0](https://explore.whatismybrowser.com/useragents/parse/?analyse-my-user-agent=yes)
 
### What did you do?

 
I opened the page above. 
 
### What did you expect to see?
 
A prominent deprecation warning next to the package description. I expect this, because the [package comment](https://github.com/googleapis/google-api-go-client/blob/v0.171.0/sql/v1beta4/sql-gen.go#L43) for the source file starts with "// Deprecated: ".
 
### What did you see instead?

No mention of the package being deprecated.

**Comment by jfrech** [comment]
> No mention of the package being deprecated.

My browser renders the deprecation paragraph at the very end of the "Overview" section, behaving analogous to e.&hairsp;g. <https://pkg.go.dev/io/ioutil>.

**Comment by johannes-riecken** [comment]
> > No mention of the package being deprecated.
> 
> My browser renders the deprecation paragraph at the very end of the "Overview" section, behaving analogous to e. g. https://pkg.go.dev/io/ioutil.

Oh, my bad, I must have missed that. I'd prefer having it more visually prominent though, as it's easy to miss.

**Comment by jfrech** [comment]
I would agree: a beige background -- as is done with deprecated functions -- around the "Deprecated:" paragraph I would welcome.
